README.md

```diff
@@ -149,195 +129,86 @@ export const lists = {
+- [Page content strings](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/pages/index.ts).
+- [Field content strings](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/fields/index.ts).
 
-When a new sprint starts - once an "in progress" ticket is ready for PR review, instead of picking up a new feature ticket, we prioritise bug fixes before picking up a new feature ticket. This is to ensure that everything works as expected and we don't end up with a large list of bugs in the backlog.
+Field IDs are stored in [constants](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/constants) rather than content strings because an ID is not a content string.
 
-Recommened: When fixing a bug, write an E2E test first and then fix it. We need a test to make sure the bug doesn't happen again.
+## DRY UI Controllers and Rendering :repeat:
 
-## Keeping page and fields separated
+To maintain consistency and avoid repetition, a common pattern is followed for controllers. Page variables and templates are defined separately. Additionally, page variable functions are used to achieve DRY (Don't Repeat Yourself) code and facilitate testing.
 
-Each page and field has content specific to that page or field. We keep these in their own files, separated from the code itself for several reasons:
-
-- To avoid having content is hard coded in the templates
-- To avoid referencing a field ID or value by hard coding it in logic or tests
-- Enforces a single source of truth
-- Provides us with an easy way to change or rename content - change it one single place
-- Provides us with an opportunity in the future to extract the content into Keystone and allow designers to edit the content themselves
-
-We have the same approach for content that is used throughout, for example buttons, links, components that are used in multiple places etc.
-
-The files can be found  here:
-
-- [global content strings](https://github.com/UK-Export-Finance/exip/tree/main-application/src/ui/server/content-strings) (buttons, links etc)
-- [page content strings](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/pages/index.ts)
-- [field content strings](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/content-strings/fields/index.ts)
-
-Note: field IDs are in [constants](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/constants) because an ID is not a content string.
-
-## DRY UI controllers and rendering
-
-Every page/controller requires the same content and links in order to meet design. For example, every page has a header, footer, service name and a back link. To save repetition, we've introduced **page variables functions** that should be used in every single controller.
```

Comment:
Are the examples of page variables etc intentionally removed? A lot is happening with these and has been confusing for new developers, which is why it was added to the README

Reply:
Apologies this has been removed by mistake, have added it back.